docs/storage/azure-storage.md

```diff
@@ -58,188 +71,149 @@ You also need to assign the following roles to the user account you are logged i
 
 Visual Studio creates a new ASP.NET Core solution that is structured to use .NET Aspire. The solution consists of the following projects:
 
-- **AspireStorage** - A Blazor project with default .NET Aspire service configurations.
-- **AspireStorage.AppHost** - An orchestrator project designed to connect and configure the different projects and services of your app. The orchestrator should be set as the startup project.
-- **AspireStorage.ServiceDefaults** - A shared class library to hold configurations that can be reused across the projects in your solution.
+- **AspireStorage**: A Blazor projects' directory that contains the following projects:
```

Comment:
I don't think we need this project to include the .Client project for webassembly - there's no interactivity anywhere, can we just make this a single project SSR Blazor app for simplicity?

Reply:
That's what comes with the Blazor Web App template. I followed the original instructions, and that's what was created - it just wasn't documented that way. I left the defaults, as detailed.

Reply:
The default doesn’t include web assembly

Reply:
Ok, so TIL that the **Interactive render mode** is sticky? Meaning at one point, I selected **Auto**, so now it defaults on my machine to **Auto**. Is that true?

Reply:
Yeah it's sticky, the first time you use the template it defaults to None for interactive render mode, and from then on it remembers what you picked, I just tried it. I just think having the .client project implies its being used for something for the tutorial when it isn't. Technically it doesn't matter which one they pick since the Home component used by the tutorial will be set up to use SSR regardless of what option they pick, but we could tell them to choose None if we want to keep it consistent.